addons/web/static/src/scss/notification.scss

```diff
@@ -0,0 +1,52 @@
+// ------------------------------------------------------------------
+// Misc. widgets
+// ------------------------------------------------------------------
+.o_notification_manager {
+    width: 300px;
+    max-width: 100%;
+
+    @include o-position-absolute($nav-link-height, 0);
+    z-index: 1100; // Bootstrap modal z-index is 1050
+
+    .o_notification {
```

Comment:
Still use of all of this below if we are using standard bootstrap toast components ?